docs/content/guides/major-pg-upgrade.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "PostgreSQL Major Upgrade"
+date:
+draft: false
+weight: 100
+---
+
+You can perform a major PostgreSQL upgrade from one version to another using PGO. Described below are the 
```

Comment:
```suggestion
You can perform a PostgreSQL upgrade from one major version to another using PGO. Described below are the 
```
⛏️ Just thinking to avoid the ambiguity of "major PostgreSQL upgrade"